core/embed/rust/src/ui/layout/obj.rs

```diff
@@ -146,10 +146,12 @@ where
         self.returned_value.as_ref()
     }
 
-    fn paint(&mut self) {
+    fn paint(&mut self) -> Result<(), Error> {
         render_on_display(None, Some(Color::black()), |target| {
             self.inner.render(target);
         });
+
+        Ok(())
```

Comment:
any reason the exception is not raised also here? means we're not testing pre-Delizia layouts?

(if that's the intention, please at least make a TODO comment)

Reply:
That was not the intention, which led to more crashes on other models, of course. :) Still WIP...